docs/index.rst

```diff
@@ -694,6 +686,57 @@ its :ref:`firstresult` in which case only the first single value (which is not
 
 .. _call_historic:
 
+Exception handling
+------------------
+If any *hookimpl* errors with an exception no further callbacks
+are invoked and the exception is packaged up and delivered to
+any :ref:`hookwrappers` before being re-raised at the hook invocation
+point:
+
+.. code-block:: python
+
+    from pluggy import PluginManager, HookimplMarker
+
+    hookimpl = HookimplMarker('myproject')
+
+    class Plugin1(object):
+        @hookimpl
+        def myhook(self, args):
+            return 1
+
+    class Plugin2(object):
+        @hookimpl
+        def myhook(self, args):
+            raise RunTimeError
```

Comment:
Nitpick: RuntimeError (the t is lowercase)

Reply:
Thanks, fixed!

Reply:
Shoot.. nice catch!